packages/core/src/analytics/integrations/__tests__/Omnitracking.test.js

```diff
@@ -372,7 +373,96 @@ describe('Omnitracking', () => {
           expect(mockLoggerWarn).not.toHaveBeenCalled();
         });
       });
+
+      describe('Event Tracking', () => {
+        describe('Interact Content', () => {
+          it('should warn if an interact content does not have required parameters', async () => {
+            const data = generateTrackMockData({
+              event: eventTypes.INTERACT_CONTENT,
+              properties: {
+                interactionType: undefined,
+              },
+            });
+            await omnitracking.track(data);
+
+            expect(mockLoggerWarn).toHaveBeenCalledWith(
+              expect.stringContaining(
+                'requires at least properties contentType',
+              ),
+            );
+          });
+
+          it('should track scroll event', async () => {
+            const data = generateTrackMockData({
+              event: eventTypes.INTERACT_CONTENT,
+              properties: {
+                interactionType: interactionTypes.SCROLL,
+                target: document.body,
+                percentageScrolled: 50,
+              },
+            });
+            await omnitracking.track(data);
+
+            expect(postTrackingsSpy).toHaveBeenCalledWith(
+              expect.objectContaining({
+                parameters: expect.objectContaining({
+                  tid: 668,
+                }),
+              }),
+            );
+          });
+        });
+
+        it('should not track any event', async () => {
```

Comment:
This test description is too generic, I think it would read best if it was something like "should not track an event when `interactionType` is scroll but target is not `document.body`". What do you think?